pkg/oc/cli/admin/migrate/scc/scc.go

```diff
@@ -278,3 +303,46 @@ func convertSccToClusterRoleBinding(scc *securityapi.SecurityContextConstraints)
 
 	return &binding, nil
 }
+
+func convertClusterRolesWithSCCRules(scc *securityapi.SecurityContextConstraints, roles *rbacv1.ClusterRoleList) (rolesToUpdate []rbacv1.ClusterRole) {
+	for roleIt := 0; roleIt < len(roles.Items); roleIt++ {
+		if convertRulesWithSCCName(&roles.Items[roleIt].Rules, scc.Name) {
+			rolesToUpdate = append(rolesToUpdate, roles.Items[roleIt])
+		}
+	} // ClusterRoles
+	return
+}
+
+func convertRolesWithSCCRules(scc *securityapi.SecurityContextConstraints, roles *rbacv1.RoleList) (rolesToUpdate []rbacv1.Role) {
+	for roleIt := 0; roleIt < len(roles.Items); roleIt++ {
+		if convertRulesWithSCCName(&roles.Items[roleIt].Rules, scc.Name) {
+			rolesToUpdate = append(rolesToUpdate, roles.Items[roleIt])
```

Comment:
I personally wouldn't use named return variable here...

Reply:
Ok, I'll fix that.